cross-env BABEL_ENV=test karma start test/karma.conf.js --single-run keep failing

## Bug Report

### Which version of ElasticJob did you use?

b38c1c5c007c072a4cd3d96c63804cd0d738c95e

### Expected behavior

Unit tests passed.

### Actual behavior

```
[INFO] 25 10 2022 14:16:45.072:ERROR [launcher]: ChromeHeadless stdout: 
[INFO] 25 10 2022 14:16:45.072:ERROR [launcher]: ChromeHeadless stderr: 
[INFO] DevTools listening on ws://127.0.0.1:9222/devtools/browser/36cac8f5-a8ac-4be7-8660-91ddabfcf353
[INFO] [1025/141644.550991:WARNING:gpu_process_host.cc(1206)] The GPU process has crashed 1 time(s)
[INFO] [1025/141644.639589:WARNING:gpu_process_host.cc(1206)] The GPU process has crashed 2 time(s)
[INFO] [1025/141644.719910:WARNING:gpu_process_host.cc(1206)] The GPU process has crashed 3 time(s)
[INFO] [1025/141644.798477:WARNING:gpu_process_host.cc(1206)] The GPU process has crashed 4 time(s)
[INFO] [1025/141644.885891:WARNING:gpu_process_host.cc(1206)] The GPU process has crashed 5 time(s)
[INFO] [1025/141644.975468:WARNING:gpu_process_host.cc(1206)] The GPU process has crashed 6 time(s)
[INFO] [1025/141644.975501:FATAL:gpu_data_manager_impl_private.cc(918)] The display compositor is frequently crashing. Goodbye.
[INFO] #0 0x55561be1c879 base::debug::CollectStackTrace()
[INFO] #1 0x55561bd79c63 base::debug::StackTrace::StackTrace()
[INFO] #2 0x55561bd8e013 logging::LogMessage::~LogMessage()
[INFO] #3 0x55561a48fdec content::GpuDataManagerImplPrivate::FallBackToNextGpuMode()
[INFO] #4 0x55561a48d7bf content::GpuDataManagerImpl::FallBackToNextGpuMode()
[INFO] #5 0x55561a4955e1 content::GpuProcessHost::RecordProcessCrash()
[INFO] #6 0x55561a496e7e content::GpuProcessHost::OnProcessCrashed()
[INFO] #7 0x55561a2f369e content::BrowserChildProcessHostImpl::OnChildDisconnected()
[INFO] #8 0x55561c74096a IPC::internal::MessagePipeReader::OnPipeError()
[INFO] #9 0x55561bf392ee mojo::InterfaceEndpointClient::NotifyError()
[INFO] #10 0x55561c745137 IPC::(anonymous namespace)::ChannelAssociatedGroupController::NotifyEndpointOfError()
[INFO] #11 0x55561c7462ed IPC::(anonymous namespace)::ChannelAssociatedGroupController::OnPipeError()
[INFO] #12 0x55561bf35796 mojo::Connector::HandleError()
[INFO] #13 0x55561bf4c35e mojo::SimpleWatcher::OnHandleReady()
[INFO] #14 0x55561bf4c638 mojo::SimpleWatcher::Context::Notify()
[INFO] #15 0x55561bf4bcb0 mojo::SimpleWatcher::Context::CallNotify()
[INFO] #16 0x55561a085019 mojo::core::WatcherDispatcher::InvokeWatchCallback()
[INFO] #17 0x55561a08499c mojo::core::Watch::InvokeCallback()
[INFO] #18 0x55561a0810ad mojo::core::RequestContext::~RequestContext()
[INFO] #19 0x55561a0781b9 mojo::core::NodeChannel::OnChannelError()
[INFO] #20 0x55561a08972b mojo::core::(anonymous namespace)::ChannelPosix::OnFileCanReadWithoutBlocking()
[INFO] #21 0x55561be32f23 base::MessagePumpLibevent::OnLibeventNotification()
[INFO] #22 0x55561bf038ad event_base_loop
[INFO] #23 0x55561be33197 base::MessagePumpLibevent::Run()
[INFO] #24 0x55561bdd5d49 base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::Run()
[INFO] #25 0x55561bdadc27 base::RunLoop::RunWithTimeout()
[INFO] #26 0x55561a301af4 content::BrowserProcessSubThread::IOThreadRun()
[INFO] #27 0x55561bdf16ff base::Thread::ThreadMain()
[INFO] #28 0x55561be2cece base::(anonymous namespace)::ThreadFunc()
[INFO] #29 0x7f1b2250bb43 <unknown>
[INFO] #30 0x7f1b2259da00 <unknown>
[INFO] 
[INFO] Received signal 6
[INFO] #0 0x55561be1c879 base::debug::CollectStackTrace()
[INFO] #1 0x55561bd79c63 base::debug::StackTrace::StackTrace()
[INFO] #2 0x55561be1c401 base::debug::(anonymous namespace)::StackDumpSignalHandler()
[INFO] #3 0x7f1b224b9520 <unknown>
[INFO] #4 0x7f1b2250da7c pthread_kill
[INFO] #5 0x7f1b224b9476 raise
[INFO] #6 0x7f1b2249f7f3 abort
[INFO] #7 0x55561be1b265 base::debug::BreakDebugger()
[INFO] #8 0x55561bd8e461 logging::LogMessage::~LogMessage()
[INFO] #9 0x55561a48fdec content::GpuDataManagerImplPrivate::FallBackToNextGpuMode()
[INFO] #10 0x55561a48d7bf content::GpuDataManagerImpl::FallBackToNextGpuMode()
[INFO] #11 0x55561a4955e1 content::GpuProcessHost::RecordProcessCrash()
[INFO] #12 0x55561a496e7e content::GpuProcessHost::OnProcessCrashed()
[INFO] #13 0x55561a2f369e content::BrowserChildProcessHostImpl::OnChildDisconnected()
[INFO] #14 0x55561c74096a IPC::internal::MessagePipeReader::OnPipeError()
[INFO] #15 0x55561bf392ee mojo::InterfaceEndpointClient::NotifyError()
[INFO] #16 0x55561c745137 IPC::(anonymous namespace)::ChannelAssociatedGroupController::NotifyEndpointOfError()
[INFO] #17 0x55561c7462ed IPC::(anonymous namespace)::ChannelAssociatedGroupController::OnPipeError()
[INFO] #18 0x55561bf35796 mojo::Connector::HandleError()
[INFO] #19 0x55561bf4c35e mojo::SimpleWatcher::OnHandleReady()
[INFO] #20 0x55561bf4c638 mojo::SimpleWatcher::Context::Notify()
[INFO] #21 0x55561bf4bcb0 mojo::SimpleWatcher::Context::CallNotify()
[INFO] #22 0x55561a085019 mojo::core::WatcherDispatcher::InvokeWatchCallback()
[INFO] #23 0x55561a08499c mojo::core::Watch::InvokeCallback()
[INFO] #24 0x55561a0810ad mojo::core::RequestContext::~RequestContext()
[INFO] #25 0x55561a0781b9 mojo::core::NodeChannel::OnChannelError()
[INFO] #26 0x55561a08972b mojo::core::(anonymous namespace)::ChannelPosix::OnFileCanReadWithoutBlocking()
[INFO] #27 0x55561be32f23 base::MessagePumpLibevent::OnLibeventNotification()
[INFO] #28 0x55561bf038ad event_base_loop
[INFO] #29 0x55561be33197 base::MessagePumpLibevent::Run()
[INFO] #30 0x55561bdd5d49 base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::Run()
[INFO] #31 0x55561bdadc27 base::RunLoop::RunWithTimeout()
[INFO] #32 0x55561a301af4 content::BrowserProcessSubThread::IOThreadRun()
[INFO] #33 0x55561bdf16ff base::Thread::ThreadMain()
[INFO] #34 0x55561be2cece base::(anonymous namespace)::ThreadFunc()
[INFO] #35 0x7f1b2250bb43 <unknown>
[INFO] #36 0x7f1b2259da00 <unknown>
[INFO]   r8: 00007f1b1d081720  r9: 0000000000000000 r10: 0000000000000008 r11: 0000000000000246
[INFO]  r12: 0000000000000006 r13: 0000000000000016 r14: 00007f1b1d0818b8 r15: 00007f1b1d0818d0
[INFO]   di: 0000000000094a28  si: 0000000000094a31  bp: 0000000000094a31  bx: 00007f1b1d084640
[INFO]   dx: 0000000000000006  ax: 0000000000000000  cx: 00007f1b2250da7c  sp: 00007f1b1d081650
[INFO]   ip: 00007f1b2250da7c efl: 0000000000000246 cgf: 002b000000000033 erf: 0000000000000004
[INFO]  trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000000
[INFO] [end of stack trace]
[INFO] Calling _exit(1). Core file will not be generated.
[INFO] 
[INFO] 25 10 2022 14:16:45.075:ERROR [launcher]: ChromeHeadless failed 2 times (cannot start). Giving up.
[INFO] 
[INFO] 
[INFO] 
[INFO] npm ERR! code ELIFECYCLE
[INFO] npm ERR! errno 1
[INFO] npm ERR! shardingsphere-elasticjob-ui-frontend@1.0.0 unit: `cross-env BABEL_ENV=test karma start test/karma.conf.js --single-run`
[INFO] npm ERR! Exit status 1
[INFO] npm ERR! 
[INFO] npm ERR! Failed at the shardingsphere-elasticjob-ui-frontend@1.0.0 unit script.
[INFO] npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
[INFO] 
[INFO] npm ERR! A complete log of this run can be found in:
[INFO] npm ERR!     /home/wuweijie/.npm/_logs/2022-10-25T06_16_45_137Z-debug.log
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for shardingsphere-elasticjob-ui 3.1.0-SNAPSHOT:
[INFO] 
[INFO] shardingsphere-elasticjob-ui ....................... SUCCESS [  1.251 s]
[INFO] shardingsphere-elasticjob-lite-ui .................. SUCCESS [  0.668 s]
[INFO] shardingsphere-elasticjob-lite-ui-frontend ......... FAILURE [03:05 min]
[INFO] shardingsphere-elasticjob-lite-ui-backend .......... SKIPPED
[INFO] shardingsphere-elasticjob-cloud-ui ................. SKIPPED
[INFO] shardingsphere-elasticjob-cloud-ui-frontend ........ SKIPPED
[INFO] shardingsphere-elasticjob-cloud-ui-backend ......... SKIPPED
[INFO] shardingsphere-elasticjob-ui-distribution .......... SKIPPED
[INFO] shardingsphere-elasticjob-ui-src-distribution ...... SKIPPED
[INFO] shardingsphere-elasticjob-lite-ui-bin-distribution . SKIPPED
[INFO] shardingsphere-elasticjob-cloud-ui-bin-distribution  SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  03:07 min
[INFO] Finished at: 2022-10-25T14:16:45+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.12.0:npm (npm run unit) on project shardingsphere-elasticjob-lite-ui-frontend: Failed to run task: 'npm run unit --https-proxy=http://127.0.0.1:8889 --proxy=http://127.0.0.1:8889' failed. org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :shardingsphere-elasticjob-lite-ui-frontend

```

--- a/shardingsphere-elasticjob-cloud-ui/shardingsphere-elasticjob-cloud-ui-frontend/package.json
+++ b/shardingsphere-elasticjob-cloud-ui/shardingsphere-elasticjob-cloud-ui-frontend/package.json
@@ -80,7 +80,7 @@
     "postcss-import": "^11.0.0",
     "postcss-loader": "^2.0.8",
     "postcss-url": "^7.2.1",
-    "puppeteer": "^1.20.0",
+    "puppeteer": "^19.1.1",
     "rimraf": "^2.6.0",
     "sass-loader": "^7.1.0",
     "semver": "^5.3.0",

--- a/shardingsphere-elasticjob-lite-ui/shardingsphere-elasticjob-lite-ui-frontend/package.json
+++ b/shardingsphere-elasticjob-lite-ui/shardingsphere-elasticjob-lite-ui-frontend/package.json
@@ -77,7 +77,7 @@
     "postcss-import": "^11.0.0",
     "postcss-loader": "^2.0.8",
     "postcss-url": "^7.2.1",
-    "puppeteer": "^1.20.0",
+    "puppeteer": "^19.1.1",
     "rimraf": "^2.6.0",
     "sass-loader": "^7.1.0",
     "semver": "^5.3.0",

